GamebookEngine/Views/Editing/Page List/PagesTableViewController.swift

```diff
@@ -72,8 +88,40 @@ class PagesTableViewController: UITableViewController {
     }
 
     override func tableView(_: UITableView, didSelectRowAt indexPath: IndexPath) {
-        guard let page = pages.item(at: indexPath.row) else { return }
-        delegate?.selectedPage(page)
+        if searching {
+            guard let filteredPage = filteredPages.item(at: indexPath.row) else { return }
+            delegate?.selectedPage(filteredPage)
+        } else {
+            guard let page = pages.item(at: indexPath.row) else { return }
+            delegate?.selectedPage(page)
+        }
+
         navigationController?.popViewController(animated: true)
     }
+
+    // MARK: Search Bar
+
+    func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
```

Comment:
I'm vaguely concerned about performance when it comes to refreshing the list view on every key stroke. Might be good to build in some sort of delay to ensure the user is done typing before performing the search?

Reply:
Are you still working on this part?

Reply:
Yes. I am still working on it. I will let you know if I get stuck on something.

Reply:
Okay, cool, just checking in! Hope you're enjoying your weekend.